build/commands/lib/config.js

```diff
@@ -476,7 +476,7 @@ Config.prototype.buildArgs = function () {
 
 Config.prototype.shouldSign = function () {
   if (this.skip_signing ||
-    this.buildConfig !== 'Release' ||
+    this.isComponentBuild() ||
```

Comment:
component builds aren't signable because it would have to sign every single shared lib

Reply:
Perhaps we should print to console if the user has asked for signing but there's another reason that we can't

Reply:
well, there's no way to ask for signing, there's only a flag to skip signing